.woodpecker/test.yml

```diff
@@ -13,7 +13,7 @@ steps:
   test-chart:
     image: quay.io/helmpack/chart-testing
     commands:
-      - git fetch origin main --unshallow --no-tags
+      - git fetch origin main --no-tags
```

Comment:
```suggestion
      - git fetch origin main --unshallow --no-tags || true
```

Reply:
If we do so, we should also add a comment why this was done :)

Reply:
sounds like a good idea